controller/proxy-injector/webhook_config_test.go

```diff
@@ -22,13 +21,12 @@ func TestCreateOrUpdate(t *testing.T) {
 		t.Fatal("Unexpected error: ", err)
 	}
 
-	trustAnchorsPath, err := factory.CATrustAnchors()
+	rootCA, err := tls.NewCA()
 	if err != nil {
-		t.Fatal("Unexpected error: ", err)
+		log.Fatalf("failed to create root CA: %s", err)
```

Comment:
This should be `t.Fatalf` to actually fail the test. Calling `log.Fatalf` will cause the test to exit immediately, but the error won't be printed, since all log output is being discarded by the `log.SetOutput(ioutil.Discard)` call on line 17 in this file.

```suggestion
		t.Fatalf("failed to create root CA: %s", err)
```

Reply:
oh indeed. Nice catch.